tools/link.py

```diff
@@ -2963,8 +2965,6 @@ def add_js_deps(sym):
       settings.ASYNCIFY_IMPORTS_EXCEPT_JS_LIBS = settings.ASYNCIFY_IMPORTS[:]
       settings.ASYNCIFY_IMPORTS += ['*.' + x for x in js_info['asyncFuncs']]
 
-  phase_calculate_system_libraries(state, linker_arguments, newargs)
-
   phase_link(linker_arguments, wasm_target, js_syms)
```

Comment:
I think you can revert this file since this will be part of the "port settings" PR?

Reply:
No this has nothing to do with port settings. It is a change that is required for using js_libraries in ports